### ISSUE TITLE
Display of Pre-publication and Visibility statuses

### ISSUE THREAD

**Issue by stevejburge** [publishpress/publishpress-statuses#82]
Can we make these more consistent?

In Quick Edit:
![Screenshot 2023-12-14 at 12 32 25 PM](https://github.com/publishpress/publishpress-statuses/assets/3868028/f7fafad2-f73b-4745-9deb-18a203a89754)

**Comment by agapetry** [comment]
@stevejburge I don't think it would be wise to put Visibility statuses in the Status dropdown, since WordPress does not put the normal Private status there.  It would be hard to see whether you're moving the post to a pre-publication Workflow status or a custom Visibility status.

**Comment by stevejburge** [comment]
@agapetry Agreed. I'm just suggesting more consistency ... 

- dropdown vs radio button
- left of screen vs right of screen etc.

**Comment by agapetry** [comment]
Okay, this will be a good v1.3 improvement.
